packages/tools/devtools/devtools-view/src/components/OpLatencyView.tsx

```diff
@@ -44,6 +51,105 @@ const useStyles = makeStyles({
 export function OpLatencyView(): React.ReactElement {
 	const styles = useStyles();
 
+	const messageRelay = useMessageRelay();
+	const [durationOutboundBatchingData, setDurationOutboundBatchingData] =
+		React.useState<GraphDataSet>({
+			graphType: "line",
+			schema: {
+				displayName: "Duration Outbound",
+				uuid: "durationOutbound",
+				xAxisDataKey: "outboundtimestamp",
+				yAxisDataKey: "durationOutboundBatching",
+			},
+			data: [],
+		});
+	const [durationNetworkData, setDurationNetworkData] = React.useState<GraphDataSet>({
+		graphType: "line",
+		schema: {
+			displayName: "Duration Network",
+			uuid: "durationNetwork",
+			xAxisDataKey: "networktimestamp",
+			yAxisDataKey: "durationNetwork",
+		},
+		data: [],
+	});
+	const [durationInboundToProcessingData, setDurationInboundToProcessingData] =
+		React.useState<GraphDataSet>({
+			graphType: "line",
+			schema: {
+				displayName: "Duration Inbound",
+				uuid: "durationInbound",
+				xAxisDataKey: "inboundtimestamp",
+				yAxisDataKey: "durationInboundToProcessing",
+			},
+			data: [],
+		});
+
+	React.useEffect(() => {
+		/**
+		 * Handlers for inbound messages.
+		 */
+		const inboundMessageHandlers: InboundHandlers = {
+			[TelemetryEvent.MessageType]: async (untypedMessage) => {
+				const message = untypedMessage as TelemetryEvent.Message;
+				const eventContents = message.data.event.logContent;
+				if (!eventContents.eventName.endsWith("OpRoundtripTime")) {
+					return true;
+				}
+
+				if (eventContents.clientType === "noninteractive/summarizer") {
+					return true;
+				}
```

Comment:
Were we hitting this condition? I don't recall that we connect the summarizer container to devtools.